api/src/main/java/org/openmrs/module/ssemrreports/reporting/library/data/evaluator/TPTCompletedDataEvaluator.java

```diff
@@ -35,9 +35,11 @@ public class TPTCompletedDataEvaluator implements PersonDataEvaluator {
 	public EvaluatedPersonData evaluate(PersonDataDefinition definition, EvaluationContext context)
 	        throws EvaluationException {
 		EvaluatedPersonData c = new EvaluatedPersonData(definition, context);
-		
-		String qry = "SELECT client_id, MID(MAX(CONCAT(encounter_datetime, is_tpt_finished)), 20) as tpt_finished FROM ssemr_etl.ssemr_flat_encounter_hiv_care_follow_up "
-		        + " WHERE  date(encounter_datetime) <= date(:endDate) GROUP BY client_id HAVING tpt_finished is not null;";
+
+		String qry = "select client_id, case when max(is_tpt_finished) = 'Yes' then 'Yes' "
```

Comment:
Query looks OK. 

Minor thing. Could you format the query in a consistent manner to what has been done with other queries such that the keywords are in upper case? It improves readability and maintainability

Reply:
Okay @moshonk On it.